example/lib/utils/utils.dart

```diff
@@ -1,16 +1,15 @@
 import 'package:dio/dio.dart';
 import 'package:flutter_dotenv/flutter_dotenv.dart';
 import 'package:likeminds_feed_flutter_core/likeminds_feed_core.dart';
-import 'package:likeminds_feed_sample/main.dart';
+import 'package:likeminds_feed_sample/globals.dart';
 
 Future<(String?, String?)> initiateUser(String uuid, String userName) async {
-  String apiKey = dotenv.get('API_KEY');
   String host = dotenv.get('HOST');
   Dio dio = Dio();
   var response = await dio.post('$host/sdk/initiate',
       options: Options(headers: {
         'Content-Type': 'application/json',
-        'x-api-key': apiKey,
```

Comment:
We can remove this file, After the latest API Security release we can use the respective showFeed method instead

Reply:
yes we can remove this util files, i have kept it for the purpose of testing the flow of `showFeedWithoutApiKey()` method if needed in future, this function can be used in callback

Reply:
Understood